README.md

```diff
@@ -0,0 +1,46 @@
+
+# Active Defense Block Lists
+
+NOTE:
+
+1.Thi Python scripts run automatically and download block lists form the internet and update on files .
+2.We have configured github action to run cron job in every to hours to update the files and push the changes into github as blob commit .
```

Comment:
What's "blob commit"?

Please use a tool like prettier to edit your markdown. 

Also, no need of "we".